app/models/foreman_tasks/task.rb

```diff
@@ -85,8 +85,26 @@ def execution_type
       delayed? ? N_('Delayed') : N_('Immediate')
     end
 
+    def get_humanized(method)
+      attr = case method
+             when :humanized_name
+               :action
+             when :humanized_input
+               :input
+             when :humanized_output
+               :output
+             else
+               nil
```

Comment:
`case` will implicitly return `nil` if none of the branches match. That's what rubocop complains about